### ISSUE TITLE
pass ga_visit_id into bouncer for windows stub installer downloads

### ISSUE THREAD

**Issue by rafrombrc** [mozilla/bedrock#9830]
## Description

When a moz.org visitor clicks on the 'download firefox' link to receive a download of the Windows stub installer, bedrock should pass the GA visit id into the bouncer API call so bouncer can emit that (along w a generated download token) into stackdriver for ultimate landing in a BigQuery data set.

---

#### Success Criteria
- [ ] Bouncer successfully receives GA visit id along with requests to initiate a Windows stub installer download

**Comment by pmac** [comment]
Looks like the new key name for this will be `visit_id` and we'll just add it to stub attribution data, right? That's what I'm getting based on the [PR for this in the stub attribution repo](https://github.com/mozilla-services/stubattribution/pull/103/).

**Comment by pmac** [comment]
@alexgibson how tough will it be do you think to grab the GA ID for users that have one and add it to the attribution data?

**Comment by alexgibson** [comment]
I took a look at this and we don't seem to have a `ga_visit_id` value in the GA JavaScript object that gets created by Google Analytics. I did a little digging and found [this article](https://www.owox.com/blog/use-cases/google-analytics-client-id/), which suggests that the client Id can be found in the `_ga` session cookie. This gives a value like `GA1.2.780885757.1610617994`. Everything after the second period is supposedly the client ID e.g. `780885757.1610617994`. Digging a little deeper, this same value also seems to be exposed in JS vai `gaGlobal.vid`.

Looking in GA under Audience -> User Explorer, I can see lists of client IDs that follow this same format, so I can only assume this is what we need.
